tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(3)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Monday at 12:00 AM'
         'Last pirmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(4)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Sunday at 12:00 AM'
         'Last sekmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(5)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Saturday at 12:00 AM'
         'Last šeštadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-03 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-03 00:00:00'))
+        // 'Last Thursday at 12:00 AM'
         'Last ketvirtadienis at 00:00',
+
         // Carbon::parse('2018-01-02 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-02 00:00:00'))
+        // 'Last Wednesday at 12:00 AM'
         'Last trečiadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-01 00:00:00')->isoFormat('Qo Mo Do Wo wo')
+        // '1st 1st 1st 1st 1st'
         '1 1 1 1 1',
+
         // Carbon::parse('2018-01-02 00:00:00')->isoFormat('Do wo')
+        // '2nd 1st'
         '2 1',
+
         // Carbon::parse('2018-01-03 00:00:00')->isoFormat('Do wo')
+        // '3rd 1st'
         '3 1',
+
         // Carbon::parse('2018-01-04 00:00:00')->isoFormat('Do wo')
+        // '4th 1st'
         '4 1',
+
         // Carbon::parse('2018-01-05 00:00:00')->isoFormat('Do wo')
+        // '5th 1st'
         '5 1',
+
         // Carbon::parse('2018-01-06 00:00:00')->isoFormat('Do wo')
+        // '6th 1st'
         '6 1',
+
         // Carbon::parse('2018-01-07 00:00:00')->isoFormat('Do wo')
+        // '7th 2nd'
         '7 1',
+
         // Carbon::parse('2018-01-11 00:00:00')->isoFormat('Do wo')
+        // '11th 2nd'
         '11 2',
+
         // Carbon::parse('2018-02-09 00:00:00')->isoFormat('DDDo')
+        // '40th'
         '40',
+
         // Carbon::parse('2018-02-10 00:00:00')->isoFormat('DDDo')
+        // '41st'
         '41',
+
         // Carbon::parse('2018-04-10 00:00:00')->isoFormat('DDDo')
+        // '100th'
         '100',
+
         // Carbon::parse('2018-02-10 00:00:00', 'Europe/Paris')->isoFormat('h:mm a z')
+        // '12:00 am cet'
         '12:00 priešpiet cet',
+
         // Carbon::parse('2018-02-10 00:00:00')->isoFormat('h:mm A, h:mm a')
+        // '12:00 AM, 12:00 am'
         '12:00 priešpiet, 12:00 priešpiet',
+
         // Carbon::parse('2018-02-10 01:30:00')->isoFormat('h:mm A, h:mm a')
+        // '1:30 AM, 1:30 am'
         '1:30 priešpiet, 1:30 priešpiet',
```

Comment:
1:30 is after midnight
so maybe `priešpiet`here was supposed to say `po vidurnakčio` === `after midnight`

Reply:
`meridiem` should be (if possible in the given language) an alternative to AM/PM (coupled with 12-hours clock system) so ideally 2 words/formulas for <12:00 and >=12:00.

If there is no such formulas in Lithuanian, we can set a more precise rule to select the moment of the day, like in Chinese:
https://github.com/briannesbitt/Carbon/blob/master/src/Carbon/Lang/zh_Hans.php#L79